src/remote.ts

```diff
@@ -828,6 +824,15 @@ export class Remote {
 		return sshConfig.getRaw();
 	}
 
+	private globalConfigs(label: string): string {
+		const vscodeConfig = vscode.workspace.getConfiguration();
+		const args: string[] = getGlobalFlags(
+			vscodeConfig,
+			path.dirname(this.storage.getSessionTokenPath(label)),
+		);
+		return args.length === 0 ? "" : ` ${args.join(" ")}`;
```

Comment:
Similar thought here about always having at least one item so the length check will never matter.

But, maybe being defensive is better, could be we change this in the future, so feel free to leave the checks.  Or if we can somehow guarantee there is at least one item using the types, maybe that would be optimal, not sure if Typescript can do that though.